pkgs/shells/bash/4.4.nix

```diff
@@ -68,10 +68,10 @@ stdenv.mkDerivation rec {
   ];
 
   # Note: Bison is needed because the patches above modify parse.y.
+  depsBuildBuild = [ buildPackages.stdenv.cc ];
   nativeBuildInputs = [bison]
     ++ optional (texinfo != null) texinfo
-    ++ optional hostPlatform.isDarwin binutils
-    ++ optional (hostPlatform != buildPlatform) buildPackages.stdenv.cc;
+    ++ optional hostPlatform.isDarwin binutils;
```

Comment:
Any idea what this `binutils` dependency is all about?

Reply:
None whatsoever.